framework/utils/src/jvmTest/resources/loginvalidation/validatePassword.csv

```diff
@@ -1,21 +1,21 @@
-"MDSLI", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one lowercase letter., Password must contain at least one number.'"
-"mphkn", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one number.'"
-"UJDLDGXGFIIFSJPJZMEFQTETU", 3 , "'Password must be between 6 and 24 characters long., Password must contain at least one lowercase letter., Password must contain at least one number.'"
-"chkxpnsdtcakxvbrynvpdvmseitjndf", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one number.'"
-"1234", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one lowercase letter.'"
-"1234567890123456789012345678090", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one lowercase letter.'"
\ No newline at end of file
+"Jei4h", 2, "'Password must be between 6 and 24 characters long., Password must contain at least one symbol.'"
+"CmXwVvZDeVXqwmcE3A23cEuyjA3", 2, "'Password must be between 6 and 24 characters long., Password must contain at least one symbol.'"
+"LqNsxaWFftJgXBkhJU", 2, "'Password must contain at least one symbol., Password must contain at least one number.'"
+"bmv1pmrkj77nt2brbz", 2, "'Password must contain at least one uppercase letter., Password must contain at least one symbol.'"
+"VKZMGAM6EJXQURJKYF", 2, "'Password must contain at least one lowercase letter., Password must contain at least one symbol.'"
+"jeki3", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one symbol.'"
+"7g6nmpppekzs68bqt4xr9gxqzysa1bneph183759", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one uppercase letter., Password must contain at least one symbol.'"
+"ZTVpVFNwbXYNvgeuLbnCEPcVhbEqjKmwksKDOsnsi", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one symbol., Password must contain at least one number.'"
+"tMymv", 3, "'Password must be between 6 and 24 characters long., Password must contain at least one symbol., Password must contain at least one number.'"
+"THISISABADPASSWORD",3,"'Password must contain at least one lowercase letter., Password must contain at least one symbol., Password must contain at least one number.'"
+"thisisalsoamalpassword",3,"'Password must contain at least one uppercase letter., Password must contain at least one symbol., Password must contain at least one number.'"
+"1234869098765", 3, "'Password must contain at least one uppercase letter., Password must contain at least one lowercase letter., Password must contain at least one symbol.'"
```

Comment:
This need to be reverted back to the original tests because they were testing EVERY combination of errors for each count of errors. 
We need a test for when the we get 1 error for missing symbols, then, combinations for 2 errors that include symbols missing, and 3 errors that include symbols missing.